src/Markup/Avalonia.Markup.Xaml/MarkupExtensions/ReflectionBindingExtension.cs

```diff
@@ -1,28 +1,34 @@
-using Avalonia.Data;
 using System;
-using Avalonia.Controls;
-using Avalonia.Data.Converters;
 using System.Diagnostics.CodeAnalysis;
-using System.ComponentModel;
-using System.Globalization;
+using Avalonia.Controls;
+using Avalonia.Data;
 
 namespace Avalonia.Markup.Xaml.MarkupExtensions
 {
     [RequiresUnreferencedCode(TrimmingMessages.ReflectionBindingRequiresUnreferencedCodeMessage)]
-    public class ReflectionBindingExtension
+    public class ReflectionBindingExtension : ReflectionBinding
```

Comment:
I find the fact that this class inherits from `ReflectionBinding` confusing as the underlying implementation isn't really used since a new instance is provided in `ProvideValue`.

Reply:
Done.

Reply:
I guess we could either:

1. Duplicate all properties of `ReflectionBinding` in `ReflectionBindingExtension` (not ideal)
2. Modify the instance of `ReflectionBindingExtension` in `ProvideValue` (to set `DefaultAnchor` and `NameScope`)

The second is possible, but I wasn't sure if it would be possible that instances would be reused?

We could also move `ProvideValue` to `ReflectionBinding` and remove `ReflectionBindingExtension`, but I'm not sure that would make the API clearer (there would be a `ProvideValue` method on `ReflectionBinding` that you shouldn't actually use).

Reply:
Let's keep it as-is, it works the same way in `CompiledBinding`.

Technically the `...BindingExtension` classes are factories are properties should probably be repeated and passed to the created binding instances.

That said, if this were types meant to be used in C# by users, the change should probably be done. But it's only for the XAML compiler, so let's keep things simple!